docker-compose.yaml

```diff
@@ -0,0 +1,30 @@
+version: '3'
+services:
+    node-engine-io-secure:
+        build:
+            context: ./ci
+        command: [ "node", "/test/engine-io-secure.js" ]
+        ports:
+            - "4202:4202"
+    node-engine-io:
+        build:
+            context: ./ci
+        command: [ "node", "/test/engine-io.js" ]
+        ports:
+            - "4201:4201"
+    node-socket-io:
```

Comment:
It might be better to move this into the .devcontainer folder.

Reply:
fixed in https://github.com/1c3t3a/rust-socketio/pull/50/commits/9d87f027635a846ac9504694c116453bea22f8b6